packages/ast-spec/project.json

```diff
@@ -16,7 +16,6 @@
       "executor": "@nx/eslint:lint",
       "outputs": ["{options.outputFile}"],
       "options": {
-        "lintFilePatterns": ["packages/ast-spec/**/*.{mts,cts,ts,tsx}"],
```

Comment:
The patterns are no longer required, the source files of the project are all eligible to be linted